core/embed/rust/src/ui/model_tt/component/passphrase.rs

```diff
@@ -23,9 +23,9 @@ pub enum PassphraseKeyboardMsg {
 pub struct PassphraseKeyboard {
     page_swipe: Swipe,
     textbox: Child<TextBox>,
-    back_btn: Child<Button>,
-    confirm_btn: Child<Button>,
-    key_btns: [[Child<Button>; KEYS]; PAGES],
+    back_btn: Child<Button<&'static str>>,
+    confirm_btn: Child<Button<&'static str>>,
+    key_btns: [[Child<Button<&'static [u8]>>; KEYS]; PAGES],
```

Comment:
We can change it to `str` but then we'd have to do `Vec<char>`*->`&[u8]` conversion before passing the phrase into `display::text()`. Since we don't support non-ASCII passphrases, using `&[u8]` directly seems like path of least resistance?

_*_ basically UTF-32

Reply:
i did a weird thing in 8670ad8f391d8f6c2bdca2b58138b22c9230cddc where `TextBox` is still `Vec<u8>` which internally guarantees that it's ASCII only, and easily converts ` -> &[u8] -> &str` for calling the modified `display::text()`

let's leave this as-is until @jpochyla has a chance to comment